jumpscale/packages/vdc_dashboard/sals/vdc_dashboard_sals.py

```diff
@@ -241,3 +241,17 @@ def _get_ingress_used_ports():
                 ports.add(path["backend"]["servicePort"])
 
     return ports
+
+
+def get_ingresstcp_used_ports():
```

Comment:
I think you need to delete this file change then rebase or merge with latest development.